src/main/java/jeryl/fyp/model/student/UniqueStudentList.java

```diff
@@ -134,4 +134,17 @@ private boolean studentsAreUnique(List<Student> students) {
         }
         return true;
     }
+
+    /**
+     * Returns unique Student if {@code students} contains the student with the specified studentId.
+     */
+    public Student getStudent(StudentId studentId) {
```

Comment:
Same here, do you think `getStudentByStudentId` is better?

Reply:
Yeap have changed it to getStudentByStudentId instead!